src/argilla/client/feedback/training/schemas.py

```diff
@@ -372,6 +372,15 @@ def for_direct_preference_optimization(
         """
         return TrainingTaskForDPO(formatting_func=formatting_func)
 
+    @classmethod
+    def for_chat_completion(
+        cls,
+        formatting_func: Callable[
+            [Dict[str, Any]], Union[None, Tuple[str, str, str, str], Iterator[Tuple[str, str, str, str]]]
+        ],
+    ) -> "TrainingTaskForChatCompletion":
```

Comment:
No need to put this type-hint between quotes, as it's not imported under the `if TYPE_CHECKING`

```suggestion
    ) -> TrainingTaskForChatCompletion:
```

Reply:
I belive it is because the TrainingTaskForChatCompletion is defined lateron in the file